configmaptocrs/main.go

```diff
@@ -105,7 +107,11 @@ func generate(w io.Writer, origin string) error {
 
 func readConfig(origin string) ([]byte, error) {
 	fp := filepath.Join(inputDirPath, origin)
-	f, err := os.Open(fp)
+	fp = filepath.Clean(fp)
+	if !strings.HasPrefix(fp, path.Clean(inputDirPath)) {
```

Comment:
we do we need this? fp is the join of inputdir and origin, so it will have inputDirPath as prefix

Reply:
actually no
I had to do it to workaround https://github.com/securego/gosec/issues/893
but still, IMHO, is a better option than masking it (with a //#nosec comment)

Reply:
again, I can add a better comment/commit msg so it'll be properly documented

Reply:
no, consider the case
inputDirPath = "/var/input"
and 
origin = "../../etc/passwd"

the join would be "/var/input/../../etc/passwd"
and the clean is "/etc/passwd" 
which doesnt have the inputDirPath prefix.

for (hopefully) obvious reasons, we want to block such cases.